<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicApp</class>
 <widget class="QMainWindow" name="MusicApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="prevSong">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSong">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playAndPause">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/recources/images/play.png</normaloff>:/recources/images/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="shutDownPC">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>shut down pc</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountOfRepeats">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="amountOfRepeates">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>set amount of repeats</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>title of song</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loopBtn">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>230</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/recources/images/loop_off.png</normaloff>:/recources/images/loop_off.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="shuffleBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/recources/images/shuffle_off.png</normaloff>:/recources/images/shuffle_off.png</iconset>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="songs">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Songs</string>
     </attribute>
     <widget class="QListWidget" name="listOfSongs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="playlists">
     <attribute name="title">
      <string>Playlists</string>
     </attribute>
     <widget class="QListWidget" name="listOfPlaylists">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>201</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listOfSongsInPlaylist">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>201</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusic">
    <property name="title">
     <string>Music</string>
    </property>
    <addaction name="addMusic"/>
    <addaction name="addPlaylist"/>
    <addaction name="removeSongsGlobally"/>
   </widget>
   <addaction name="menuMusic"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addMusic">
   <property name="text">
    <string>Add music </string>
   </property>
  </action>
  <action name="addPlaylist">
   <property name="text">
    <string>Create playlist</string>
   </property>
  </action>
  <action name="removeSongsGlobally">
   <property name="text">
    <string>Remove songs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
